logstash-output-nagios.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-nagios'
-  s.version         = '1.0.0'
+  s.version         = '1.1.0'
```

Comment:
1.0.1, instead of 1.1.0, I don't think the changes done in this PR requires us to bump the minor version.